valkeyotel/README.md

```diff
@@ -29,6 +35,25 @@ func main() {
         panic(err)
     }
     defer client.Close()
+
+    // Basic usage
+    ctx := context.Background()
+    client.DoCache(ctx, client.B().Get().Key("mykey").Cache(), time.Minute)
+
+    // Add custom labels to cache metrics using CacheLabeler
+    bookLabeler := &valkeyotel.CacheLabeler{}
```

Comment:
Use `LabelerFromContext` to create the labeler and store it into a new ctx if it doesn't exists in the old ctx.